drivers/wifi/siwx917/siwx917_wifi.h

```diff
@@ -17,6 +17,7 @@
 struct siwx917_dev {
 	struct net_if *iface;
 	sl_mac_address_t macaddr;
+	uint16_t max_bss_cnt;
```

Comment:
The name of this variable does not allow to understand what it does. Rename to `scan_max_bss_cnt` and place it beside `scan_res_cb`.